scmail.py

```diff
@@ -119,23 +121,34 @@ def register(ctx):
     else:
         print('Registration fail.\nError is: {}'.format(res['error']))
 
+    # Register
+    r = requests.post("http://127.0.0.1:8080/register/")
```

Comment:
Nit: I would recommend defining the url as a global that can be picked up from an environment variable or given as an option to the cli. Something like this:
```python
from os import environ

# Default to localhost if url is not given
SECUREMAILBOX_URL = environ.get("SECUREMAILBOX_URL", "http://localhost:8080")

# Requests could then look like this
req = requests.post(f"{SECUREMAILBOX_URL}/register")
```